webapp/portlet/src/main/webapp/vue-apps/profile-settings/components/menu/ProfileSettingsActionMenu.vue

```diff
@@ -91,7 +91,7 @@ export default {
   },
   methods: {
     edit(){
-      this.$root.$emit('open-settings-edit-drawer', this.setting);
+      this.$root.$emit('open-settings-edit-drawer', JSON.parse(JSON.stringify( this.setting)));
```

Comment:
Could you give more details why we need to strigify then parse the JS Object ?

Reply:
to avoid using the same object (the changes will be bidirectional) we must parse it to a new JSON object

Reply:
@GouadriaHanen I see , then you just wanted to create a copy of the setting object with no reference to objects linked to the original one. It would be helpful if you put more details about your solution and the cause in the PR description and